pandas/_libs/tslibs/nattype.pyx

```diff
@@ -372,7 +372,7 @@ class NaTType(_NaT):
     1         NaT
     """
 
-    __module__ = "pandas"
+    __module__ = "pandas.api.typing"
```

Comment:
I get wanting NatType in the api,typing namespace, but do t we want pd.NaT in the pandas namespace? Same for pd.NA above